### PR TITLE
Add `our` style changes to toast messages.  BIDS-3206.

### PR DESCRIPTION
**Pull request by benji-bitfly**
- Add necessery new colors and variable to follow the figma design.  BIDS-3206

Screenshot from 2024-07-31 10-16-29

### CODE REVIEW

**Comment by benji-bitfly** [comment]
**UXUI approved** :heavy_check_mark:

**Review by MauserBitfly — COMMENTED** [review]
Nice job, left some comments

**Comment by benji-bitfly** [comment]
@MauserBitfly thank you :) Adjusted code for the comments.

File: frontend/assets/css/prime.scss [hunk 1]
```diff
@@ -225,93 +225,107 @@
 
 // Toast
 // https://primevue.org/toast/
-// TODO: apply our styles
 .p-toast {
   opacity: 1;
+
+  .p-toast-message {
+    margin: 0 0 1rem 0;
```
[the remark this answers is not in the document]
Comment (MauserBitfly) [reply frontend/assets/css/prime.scss]: Sorry my comment was not clear enought, please update ALL rem values in the toast styles with px ;-)

**Review by MauserBitfly — APPROVED** [review]
LGTM

**Pull request merged**